gltf/write.cpp

```diff
@@ -899,15 +911,18 @@ void writeTexture(std::string& json, const cgltf_texture& texture, cgltf_data* d
 {
 	if (texture.image)
 	{
+		append(json, "\"sampler\":");
```

Comment:
sampler is optional and thus can be null: https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#reference-texture

Reply:
Added check for null sampler